pkg/pb/orders.proto

```diff
@@ -28,8 +28,10 @@ message OrderLimit {
     // satellite who issued this order limit allowing orderer to do the specified action
     bytes satellite_id = 2 [(gogoproto.customtype) = "NodeID", (gogoproto.nullable) = false];
     // uplink who requested or whom behalf the order limit to do an action
-    bytes uplink_id = 3 [(gogoproto.customtype) = "NodeID", (gogoproto.nullable) = false];
-    // storage node who can reclaim the order limit specified by serial
+    bytes uplink_id = 3 [(gogoproto.customtype) = "NodeID"]; // deprecated
```

Comment:
Would it be a huge diff to change the name to `deprecated_uplink_id`? It may also help with being sure that we've stopped using it in the appropriate spots.

Reply:
done